### PR TITLE
netcdf-fortran: update to 4.5.3

### PR DESCRIPTION
**Pull request by remkos**
Simple version and checksum update

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.5
Xcode 11.5

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/netcdf-fortran/Portfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ PortGroup                   github 1.0
 #        I/O feature disabled. Abort.
 mpi.enforce_variant         netcdf
 
-github.setup                Unidata netcdf-fortran 4.5.2 v
+github.setup                Unidata netcdf-fortran 4.5.3 v
 revision                    2
```
Comment (mf2k) [inline science/netcdf-fortran/Portfile:23]: The `revision` needs to be set back to 0 when increasing the `version`.

Comment (remkos) [reply science/netcdf-fortran/Portfile]: Thanks, forgot. Done

**Comment by mf2k** [comment]
Squash the commits in 1 and use the message from the first one.

**Comment by remkos** [comment]
Squashed into one. Finally figured out how to do that.

**Comment by remkos** [comment]
Removed superfluous line in commit message, as also requested in case of #7737 
Can someone merge? The owner of this package, as in other cases (e.g. #7648), does not respond to any PRs.

**Pull request merged**